qrenderdoc/Windows/TextureViewer.ui

```diff
@@ -656,6 +689,9 @@
     </item>
     <item>
      <widget class="QToolButton" name="channelRed">
+      <property name="toolTip">
+       <string>Show Red</string>
```

Comment:
The C# tooltips for these also had a little hint for a useful shortcut, in case someone sees it:

> Show Red (Right click to toggle solo).